saleor/core/models.py

```diff
@@ -127,3 +127,23 @@ class Job(models.Model):
 
     class Meta:
         abstract = True
+
+
+class EventPayload(models.Model):
+    payload = JSONField(default=dict, encoder=CustomJsonEncoder)
+
+
+class EventTask(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    event_payload = models.ForeignKey(
+        EventPayload,
+        related_name="event_payloads",
```

Comment:
Are you sure that the related_name here is correct?
To fetch all EventPayload which are assigned to EventTaskID == 1, I will have to create query:
```
EventPayload.objects.filter(event_payloads__id=1)
```
related_name describes the connection from EventPayload -> EventTask, so it should be something related with event_task.
```
EventPayload.objects.filter(event_task__id=1)
```

For example ProductVariant.
It has field `product` which has `related_name=variants` (not `products`)
and you can do `Product.objects.filter(variants__id=1)`